src/interfaces.py

```diff
@@ -863,7 +863,7 @@ def dbg_get_trace_detailed(self, input, nesting) -> List[str]:
         pass
 
     @abstractmethod
-    def get_taints(self, inputs, nesting) -> List[InputTaint]:
+    def get_ctraces_taints(self, inputs, nesting) -> tuple[List[CTrace], List[InputTaint]]:
```

Comment:
`tuple` -> `Tuple`

Reply:
Would it make sense to rename it to something like `trace_test_case_with_taints`? to highlight that it's a version of `trace_test_case`